### ISSUE TITLE
Any way to change locale at the moment?

### ISSUE THREAD

**Issue by ctrinh98** [rinukkusu/simple_moment#2]
In vanilla moment.js you can change the locale to say “3m” instead of “3 minutes”, “8s” instead of “8 seconds”, etc. by manually redefining the locale function in your JavaScript code. Is there a way to do that here or perhaps in the future?

**Comment by rinukkusu** [comment]
Will look into it! 👍

**Comment by rinukkusu** [comment]
Hi @ctrinh98,

I've implemented that feature -  you can now add your own locale or extend the existing english locale if you want to:

```dart
class ShortLocaleEn extends LocaleEn {
  String get seconds => '%is';

  String get aMinute => '%im';
  String get minutes => '%im';

  String get anHour => '%ih';
  String get hours => '%ih';

  String get aDay => '%id';
  String get days => '%id';
}
```

To set the locale for all usages of `Moment`:

```dart
Moment.setLocaleGlobally(new ShortLocaleEn());
```

To set the locale only for the current instance of `Moment`:

```dart
var moment = new Moment.now().locale(new ShortLocaleEn());
```

Hope, this works for you! :)

**Comment by ctrinh98** [comment]
Thank you so much!! I can't express how incredible that was -- thank you for responding and implementing my request so quickly! This is perfect for what I want to do!

One question -- if I wanted to stop the IdentifierPosition from displaying, how would I set that? I tried writing the following but I keep getting this as an error.

> Invalid override. The type of 'ShortLocaleEn.future position' ('() -> IdentifierPosition') isn't a subtype of 'LocaleEn.future position' ('() -> IdentifierPosition').


```
class ShortLocaleEn extends LocaleEn {
  String get seconds => '%is';

  String get aMinute => '%im';
  String get minutes => '%im';

  String get anHour => '%ih';
  String get hours => '%ih';

  String get aDay => '%id';
  String get days => '%id';
  
  IdentifierPosition get futurePosition => IdentifierPosition.dontDisplay;
}
```
Again, thank you so much! This is really driving forward my enthusiasm with this Flutter project, hahaha! :)

**Comment by rinukkusu** [comment]
Whoops, I may have forgotten to export `IdentifierPosition` - please remove your `'package:simple_moment/src/identifier_position.dart'` import - this type should come now from the main Import as well! :)

It's corrected in version 0.1.1 I just published.

Also thanks for using and improving this package!! :)

**Comment by ctrinh98** [comment]
Works perfectly!! Thank you so much for responding to all my requests so quickly and for making such a useful and great package! You're the best! :)
